### PR TITLE
re-add package json

### PR DESCRIPTION
**Pull request by mihaiplesa**
## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+      "chalk": "^2.4.2",
+      "commander": "^2.9.0",
+      "fs-extra": "^8.0.1",
+      "log-update": "^3.2.0"
+    },
+    "devDependencies": {
+      "ip": "^1.1.5",
+      "jest": "^24.8.0"
+    }
+  }
```
Comment (linhkikuchi) [inline package.json]: Maybe add EOL here to be consistent with other package.json

Comment (mihaiplesa) [reply package.json]: done

File: package.json [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+            "src/chrome_frame/tools/test/reference_build/chrome_win": "%None%",
+            "src/chrome/tools/test/reference_build/chrome": "%None%",
+            "src/chrome/tools/test/reference_build/chrome_linux": "%None%",
+            "src/chrome/tools/test/reference_build/chrome_mac": "%None%",
+            "src/chrome/tools/test/reference_build/chrome_win": "%None%"
+          }
+        },
+        "brave-core": {
+          "dir": "src/brave",
+          "branch": "master",
```
Comment (linhkikuchi) [inline package.json]: should this be `"branch": "0.71.x",`?

Comment (mihaiplesa) [reply package.json]: oops my bad ,that should be 0.72.x

**Pull request merged**